pkg/policy/policiesSchema.json

```diff
@@ -0,0 +1,78 @@
+{
+  "type": "object",
+  "properties": {
+    "apiVersion": {
+      "type": "string",
+      "enum": [
+        "v1"
+      ]
+    },
+    "message": {
+      "type": "string",
+      "enum": [
+        "Should have exact one policy set as default",
+        "All policy names should be unique",
+        "All rule identifiers should be unique"
+      ]
+    },
+    "policies": {
+      "type": "array",
+      "minItems": 1,
+      "items": {
+        "type": "object",
+        "required": [
+          "name"
+        ],
+        "properties": {
+          "name": {
+            "type": "string"
+          },
+          "isDefault": {
+            "type": "boolean"
+          },
+          "message": {
+            "type": "string"
+          },
+          "rules": {
+            "items": {
+              "type": "object",
+              "properties": {
+                "identifier": {
+                  "type": "string",
+                  "pattern": "^([A-Z0-9])+(_?[A-Z0-9])*$"
+                }
+              },
+              "required": [
+                "identifier",
+                "messageOnFailure"
+              ]
+            }
+          }
+        }
+      }
+    },
+    "customRules": {
+      "type": "array",
+      "items": {
+        "identifier": "string",
```

Comment:
we need to check that identifier is not in default rules, this is from backend:
![Screen Shot 2022-03-16 at 14 48 42](https://user-images.githubusercontent.com/39004075/158593340-12253834-9cc4-4164-8fe5-09a0ac41e116.png)

Reply:
I think this is one of the abilities that we are losing due to JSON Schema limitations.

https://github.com/json-schema-org/json-schema-spec/wiki/Scope-of-JSON-Schema-Validation